include/mbedtls/mbedtls_config.h

```diff
@@ -2010,6 +2010,9 @@
  * Enable the multi-precision integer library.
  *
  * Module:  library/bignum.c
+ *          library/bignum_core.c
+ *          library/bignum_mod_raw.c
+ *          library/bignum_mod.c
```

Comment:
(v minor) should `bignum_mod.c` come before `bignum_mod_raw.c`? (and in `CMakeLists.txt`)

(it seems that with the PSA files we put `psa_crypto.c` before `psa_crypto_aead.c`, etc)